docker-compose.prod.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.8'
+version: '2.39.4'
```

Comment:
This threw an `invalid version` error for me when I attempted to run `docker-compose up`, but ran successfully when using basic docker commands.

Non-blocking, just a note. Was there a specific reason for the downgrade?